### ISSUE TITLE
How can I get history data from dataloader (example/agent_motion_prediction.ipynb)

### ISSUE THREAD

**Issue by hayoung-kim** [lyft/l5kit#94]
Hello, I wonder how can I get history position and yaw of the vehicles from `dataloader`. 
In `agent_motion_prediction` example, the number of the history frames can be set at `agent_motion_config.yaml` file. 
However, when I get data from `dataloader` built from `build_dataloader` function, I cannot find history data.

Here is a code:
```
>>> tr_it = iter(train_dataloader)
>>> data = next(tr_it)

>>> print(data.keys())
dict_keys(['image', 'target_positions', 'target_yaws', 
  'target_availabilities', 'world_to_image', 'track_id', 
  'timestamp', 'centroid', 'yaw', 'extent'])
```

and here are configurations (`agent_motion_config.yaml`)
```
###################
## Model options
model_params:
  model_architecture: "resnet50"

  history_num_frames: 3
  history_step_size: 1
  history_delta_time: 0.1

  future_num_frames: 50
  future_step_size: 1
  future_delta_time: 0.1
```

How can I get history data?
Thanks!

**Comment by lucabergamini** [comment]
Hi @hayoung-kim , we don't use history values explicitly right now (they are consumed by the rasterisers) so, differently from future values which are requested for a standard training, we don't export them. 

However, it should be fairly easy to support this by calling `_create_targets_for_deep_prediction` in `agent_sampling.py` with history information. 
If you start a PR to support it, I'm happy to review it :)

**Comment by lucabergamini** [comment]
#96 has now been merged in master and will be included in `1.0.6` release
